flexmeasures/data/services/users.py

```diff
@@ -105,13 +108,24 @@ def create_user(  # noqa: C901
             "User with username %s already exists." % username
         )
 
+    # check if we can link/create an account
+    if account_name is None:
+        raise InvalidFlexMeasuresUser("Cannot create user without knowing account_name")
```

Comment:
This error message can be more informative. I think a user may interpret `account_name` as the user's account name, which might lead to confusion.